src/swagger-specs/segmentation.yaml

```diff
@@ -1035,8 +1034,7 @@ paths:
         - $ref: '#/components/parameters/x-sandbox-name'
         - name: x-request-id
           in: header
-          description: A unique ID must be used per request.
-          required: true
+          description: A unique ID can be used per request.
```

Comment:
I would add 'that' and a reason. Why would you use it? I believe for tracking purposes.
"A unique ID that can be used per request for tracking purposes."